tekton/publish.yaml

```diff
@@ -189,9 +191,9 @@ spec:
         IMAGES="${IMAGES} ${IMAGES_PATH}/cmd/${cmd}:$(params.versionTag)"
       done
 
-      # Parse the built images from the release.yaml generated by ko
+      # Parse the built images from the minimal-release.yaml generated by ko
       koparse \
-        --path $OUTPUT_RELEASE_DIR/release.yaml \
+        --path $OUTPUT_RELEASE_DIR/minimal-release.yaml \
```

Comment:
@vdemeester is the one to ask. =)

Reply:
So, today, we are shipping `release.yaml` and `resolvers.yaml`. This would make it so that `release.yaml` has the resolvers by default, but I would like to keep a release.yaml that doesn't have those as well (and thus also keep resolvers.yaml).

Maybe instead of using `ko resolve` multiple time, we could use `yq` to generate new files with the thing we want to filter out. (And this would also be true for the `-t` as we could remove the tag part from it using `yq`).

Reply:
My question was not about `koparse`, not `ko resolve` actually.
The `ko resolve` calls should be fine, since the built images will be in the cache so it won't actually build the same image more than once. 

The `koparse` is not a big deal, but I would prefer a simpler script / logic where possible.
The split was done because of v0.40, now we could revert that and go back to one `koparse` invocation and fewer params in the pipeline/task.

Reply:
yes :)

Reply:
I've opened #5607 for that discussion.